openshift-examples/keycloak-https.json

```diff
@@ -42,6 +43,41 @@
             "value": "H2",
             "required": true
         },
+        {
+            "displayName": "DB ADDR",
+            "description": "Specify the name of the database pod if you are going to use an external DB. The name of the DB pod should not be 'mysql' due to an keycloak internal bug",
+            "name": "DB_ADDR",
+            "from": "maximum length of 253 lower case alphanumeric characters, -, and .",
+            "value": "",
+            "required": false
+        },
+        {
+            "displayName": "DB Port",
+            "description": "DB port (default ports for H2 = 9123, MYSQL and MARIADB = 3306, POSTGRES = 5432",
+            "name": "DB_PORT",
+            "value": "9123",
+            "required": true
```

Comment:
DB_PORT is not required. It will default to the default port by the DB vendor

Reply:
this caused the issue for me, if not set the init script set it to a wrong value